test/e2e/framework/ginkgowrapper.go

```diff
@@ -186,24 +181,21 @@ func (f *Framework) Describe(text string, args ...interface{}) bool {
 //
 // Text and arguments may be mixed. The final text is a concatenation
 // of the text arguments and special tags from the With functions.
-func Context(text string, args ...interface{}) bool {
-	return registerInSuite(ginkgo.Context, text, args)
+func Context(args ...interface{}) bool {
+	return registerInSuite(ginkgo.Context, args)
 }
 
 // Context is a shorthand for the corresponding package function.
-func (f *Framework) Context(text string, args ...interface{}) bool {
-	return registerInSuite(ginkgo.Context, text, args)
+func (f *Framework) Context(args ...interface{}) bool {
+	return registerInSuite(ginkgo.Context, args)
 }
 
 // registerInSuite is the common implementation of all wrapper functions. It
 // expects to be called through one intermediate wrapper.
-func registerInSuite(ginkgoCall func(text string, args ...interface{}) bool, text string, args []interface{}) bool {
+func registerInSuite(ginkgoCall func(test string, args ...interface{}) bool, args []interface{}) bool {
```

Comment:
where is test used now?

Reply:
It's just part of the type definition of `ginkgoCall`, so the name doesn't matter. But I didn't mean to change it - fat-fingered. Let me restore `text`.

Reply:
Done: https://github.com/kubernetes/kubernetes/compare/56983eaaba64932e504fbcf55a054ae83a3a64ee..6f2591866ed8a479743ca5cb54eaf09a3a3139d4